docs/reference/cli/resource/list.md

```diff
@@ -45,26 +45,28 @@ dsc resource list
-Microsoft.Windows/WindowsPowerShell         Adapter   0.1.0    gs-t---                  Resource adapter to classic DSC Powershell resources in Windows PowerShell.
-Microsoft/OSInfo                            Resource  0.1.0    g----e-                  Returns information about the operating system.
-Microsoft/Process                           Resource  0.1.0    gs-t-e-                  Returns information about running processes.
-Test/Delete                                 Resource  0.1.0    g---d--
-Test/Echo                                   Resource  0.1.0    gs-t---
-Test/Exist                                  Resource  0.1.0    gsx----
-Test/Sleep                                  Resource  0.1.0    gs-t---
-Test/TestGroup                              Adapter   0.1.0    g------
-Test/Trace                                  Resource  0.1.0    gs-t---
+Type                                        Kind      Version  Caps      RequireAdapter  Description
+--------------------------------------------------------------------------------------------------------------------------------------------------------------------
+Microsoft.DSC.Transitional/RunCommandOnSet  Resource  0.1.0    gs------                  Takes a single-command line to execute on DSC set operation
+Microsoft.DSC/Assertion                     Group     0.1.0    gs--t---                  `test` will be invoked for all resources in the supplied configuration.    
+Microsoft.DSC/Group                         Group     0.1.0    gs--t---                  All resources in the supplied configuration is treated as a group.
+Microsoft.DSC/Include                       Import    0.1.0    -------r                  Allows including a configuration file contents into current configuration. 
+Microsoft.DSC/Parallel                      Group     0.1.0    gs--t---                  All resources in the supplied configuration run concurrently.
+Microsoft.DSC/PowerShell                    Adapter   0.1.0    gs--t-e-                  Resource adapter to classic DSC Powershell resources.
+Microsoft.Windows/RebootPending             Resource  0.1.0    g-------                  Returns info about pending reboot.
+Microsoft.Windows/Registry                  Resource  0.1.0    gs---d--                  Manage Windows Registry keys and values
+Microsoft.Windows/WMI                       Adapter   0.1.0    g-------                  Resource adapter to WMI resources.
+Microsoft.Windows/WindowsPowerShell         Adapter   0.1.0    gs--t---                  Resource adapter to classic DSC Powershell resources in Windows PowerShell.
+Microsoft/OSInfo                            Resource  0.1.0    g-----e-                  Returns information about the operating system.
+Microsoft/Process                           Resource  0.1.0    gs--t-e-                  Returns information about running processes.
+Test/Delete                                 Resource  0.1.0    g----d--
```

Comment:
Should probably remove the Test resources from the example as they should only show up if you build the repo and the packages shouldn't contain them